pkg/cmd/server/plugin/plugin.go

```diff
@@ -87,8 +87,13 @@ func NewCommand(f client.Factory) *cobra.Command {
 					clientset, err := f.KubeClient()
 					cmd.CheckError(err)
 
-					action, err = backup.NewServiceAccountAction(logger, clientset.RbacV1().ClusterRoleBindings())
+					crbLister, err := backup.NewClusterRoleBindingLister(logger, clientset)
 					cmd.CheckError(err)
+
+					if crbLister != nil {
```

Comment:
What happens if crbLister **is** null (i.e. RBAC is not enabled on this cluster)?  We still go through the code paths of setting up an action item plugin and serving it - seems like there may be a problem there.

Reply:
Good catch - I had intended to address this and forgot.

Would it make sense to check if the action is nil before adding to the plugin configuration? Or maybe move the NewClusterRoleBindingLister logic into the NewServiceAccount action function?

Also, do we support clusters without RBAC? I didn't think we did, but even if that's the case this behavior needs to be fixed.

Reply:
I don't know if we've ever really tried on clusters without RBAC.  Hopefully shouldn't be much of a use case going fwd, but who knows.

I'd say it makes sense to always have a non-nil action.  We can move the `NewClusterRoleBindingLister` into the `NewServiceAccount` func.  If no API group is found, just leave the item action's `clusterRoleBindings` field as nil, and return the action/no error.  Then `Execute()` becomes a no-op (more or less) in that scenario.

Reply:
Sorry I didn't notice this earlier

Reply:
np, better now than after tagging!